stock_picking_invoice_link/views/account_invoice_view.xml

```diff
@@ -3,7 +3,7 @@
     <!-- Copyright 2013-2014 Alexis de Lattre <alexis.delattre@akretion.com>
          License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
     <record id="invoice_form" model="ir.ui.view">
```

Comment:
```suggestion
    <record id="view_move_form" model="ir.ui.view">
```

Reply:
this record id already exists in: https://github.com/OCA/stock-logistics-workflow/blob/14.0/stock_picking_invoice_link/views/stock_view.xml#L24
so i get:

[W7902(duplicate-xml-record-id), ]  Duplicate xml record id "data/view_move_form_noupdate_0" in views/account_invoice_view.xml:5

i added "_inherit" postfix to avoid issues.